core/src/main/java/com/google/zxing/multi/GenericMultipleBarcodeReader.java

```diff
@@ -86,8 +86,8 @@ private void doDecodeMultiple(BinaryBitmap image,
       return;
     }
     boolean alreadyFound = false;
-    for (Result existingResult : results) {
-      if (existingResult.getText().equals(result.getText())) {
+    for (Result result : results) {
```

Comment:
NATURALIZE is a framework that solves the coding convention inference problem for local conventions, offering suggestions to increase the stylistic consistency of a codebase.

Yes, there are more suggestions from Naturalize, you can go the website and download the tools available.

http://groups.inf.ed.ac.uk/naturalize/

Sorry about the compilation error, my mistake.